src/utils/retry_utils/src/retry_utils.c

```diff
@@ -259,5 +260,18 @@ void ReadRetryParamsArrayFromAgentConfigJson(
             value = DEFAULT_ENR_REQ_OP_MAX_JITTER_PERCENT;
         }
         params->maxJitterPercent = (double)value;
+
+        //
+        // Get the update polling interval retry setting, or default.
+        //
+        if (!ADUC_JSON_GetUnsignedIntegerField(
```

Comment:
The change to ADUC_JSON_GetUnsignedIntegerField() does not allow zero but if I am a customer and I want to eliminate retires it would make sense for me to configure the field with a zed value. 

Same for all of the ones below.